tests/kuttl/tests/keystone_scale/04-assert.yaml

```diff
@@ -0,0 +1,63 @@
+#
+# Check for:
+#
+# - 1 KeystoneAPI CR
+# - 1 Pods for KeystoneAPI CR
+#
+
+apiVersion: keystone.openstack.org/v1beta1
+kind: KeystoneAPI
+metadata:
+  finalizers:
+  - KeystoneAPI
+  name: keystone
+  namespace: openstack
+spec:
+  adminProject: admin
+  adminRole: admin
+  adminUser: admin
+  containerImage: quay.io/tripleozedcentos9/openstack-keystone:current-tripleo
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: keystone
+  debug:
+    bootstrap: false
+    dbSync: false
+    service: false
+  passwordSelectors:
+    admin: AdminPassword
+    database: KeystoneDatabasePassword
+  preserveJobs: false
+  region: regionOne
+  replicas: 1
+  resources:
+    requests:
+      cpu: "1.0"
+      memory: 500Mi
+  secret: osp-secret
+status:
+  apiEndpoint:
+    admin: http://keystone-admin-openstack.apps-crc.testing
+    internal: http://keystone-internal-openstack.apps-crc.testing
+    public: http://keystone-public-openstack.apps-crc.testing
+  databaseHostname: openstack
+  readyCount: 1
+---
+# We can't provide 3 separate Pod YAMLs to match against because we don't know
```

Comment:
same could check the replicas/readyReplicas of the deployment?

Reply:
Yes, we could check the deployment if we want to use that as the source of truth for the state of the pods.  I would assume the states are always consistent between the two given that it's such a core k8s paradigm.

Reply:
Done